http/src/v0/root_files/add.rs

```diff
@@ -143,15 +143,7 @@ where
             let content_type = field.content_type().map_err(AddError::Header)?;
 
             let next = match content_type {
-                "application/octet-stream" => {
-
-                    // files are of the form "file-{1,2,3,..}"
-                    let _ = if field_name != "file" && !field_name.starts_with("file-") {
-                        Err(AddError::UnsupportedField(field_name.to_string()))
-                    } else {
-                        Ok(())
-                    }?;
```

Comment:
not sure if this is a good idea to be removed. I mean if you think of a html form which is pretty much the basis for http multipart and fields with different names kinda should be different things so I'd like to retain some validation OR targeting here. Or are the go-ipfs and js-ipfs using random field names here? I'd rather see them "enumerated" here.

my comment is a bit off here, I think js-ipfs-http-client does the file-N.